bio/merqury/wrapper.py

```
@@ -8,40 +8,45 @@
-        " {snakemake.input.fasta}"
+        " {fasta}"
         " {out_prefix}"
-        " > {log_tmp} 2>&1"
+        " {log}"
     )
 
+    ### Copy files to final destination
+    def save_output(out_prefix, ext, cwd, file):
+        if file is None:
+            return 0
+        src = f"{out_prefix}{ext}"
```

Ruff

Local variable `src` is assigned to but never used

Remove assignment to unused variable `src`

(F841)